### ISSUE TITLE
Assertion `PureOperatorBuiltins == false' failed

### ISSUE THREAD

**Issue by kristerw** [KhronosGroup/glslang#124]
`glslangValidator` asserts
```
glslangValidator: /home/kristerw/glslang/glslang/MachineIndependent/ParseHelper.cpp:1539: void glslang::TParseContext::nonOpBuiltInCheck(const glslang::TSourceLoc&, const glslang::TFunction&, glslang::TIntermAggregate&): Assertion `PureOperatorBuiltins == false' failed.
Abort (core dumped)
```
when compiling the shader
```
#version 130
void main(){
        gl_Position = ftransform();
}
```
using the command line
```
glslangValidator -G bug.vert
```

**Comment by johnkslang** [comment]
Will look at preventing the assert, but it is also unlikely that glslang will support versions of GLSL for SPIR-V that are older than when ftransform() was deprecated, so the likely practical result is that you probably don't want to use ftransform() when making SPIR-V.

ftransform() was deprecated by #version 140, with the advice "Use invariant outputs instead".

**Comment by johnkslang** [comment]
This is fixed by f6eae2a54ae148ce35ba2c7eb9b4dd587e399170.
